### ISSUE TITLE
Autohoming Settings ignored in OATCONTROL SETTINGS

### ISSUE THREAD

**Issue by illigtr** [OpenAstroTech/OpenAstroTracker-Desktop#53]
I am trying to determine under what conditions the following settings are active:
- RA Start Direction
- RA Search Distance
- DEC Start Direction
- DEC Search Distance

Under numerous tests I simply cannot get any of these settings to actually work.

My Configuration:

    MKS 1.21 with 3 Triminiac 2209 V3.0 with UART configuration. (works flawlessly). Requires special diagonal jumper.
    Firmware version V1.13.5 (latest!)
    OAT 3D printed with aluminium base and modified RA and DEC mounts with belt tensioners
    1.8 NEMA 17 steppers for RA and DEC,
    MPU4050 and GPS
    RA and DEC Hall sensors for auto-homing
    DF Robot LCD (16x2) Display with keypad
    modified 28BYJ-48 stepper for DIY focuser
    Pi4 8GB
        Bookworm OS
        running latest INDI drivers (including OpenAstroTeck Mount, and LibCamera)
        Minicom and Python scripts to talk to the MKS for manual adjustments
        5Ghz WiFi
        Pi HQ camera mounted on custom Newtonion scope (LibCamera)
    Windows 11 PC running latest version of Kstars and EKOS and connected via 5 Ghz WiFi to Pi4

In  Configuration_local.hpp I have set DEC and RA limits. I have also defined autohoming

```C
// Define limits for RA... 
#define RA_LIMIT_LEFT     5.5f
#define RA_LIMIT_RIGHT    6.5f
#define RA_TRACKING_LIMIT 6.75f // can't quite get to 7h...

// Define DEC limits
#define DEC_LIMIT_UP   90 // degrees from Home
#define DEC_LIMIT_DOWN 60 // degrees from Home

// Auto Homing addons 
#define USE_HALL_SENSOR_RA_AUTOHOME 1 
#define RA_HOMING_SENSOR_SEARCH_DEGREES 30
#define RA_HOMING_SENSOR_ACTIVE_STATE HIGH
#define USE_HALL_SENSOR_DEC_AUTOHOME 1 
#define DEC_HOMING_SENSOR_SEARCH_DEGREES 60
#define DEC_HOMING_SENSOR_ACTIVE_STATE HIGH

```

I have determined that the RA and DEC limits are HARD limits, regardless of any settings in the auto-homing section.
I am attempting to be able to autohome my OAT without being physically near it. Reset always causes my camera/telescope to point straight up due to the weight of the camera/telescope, thus the need to autohoming.

So which settings have precedence? Configuration_local? How does one change RA/DEC start directions then?

I have included a photo of the particular hall sensor I am using for debugging. It includes a sensitivity adjustment was well as a power LED and a HIGH/LOW sensor output LED... useful for debugging. The sensors and magnets are temporarily held in place with double stick foam tape.

![Hall Sensors](https://github.com/OpenAstroTech/OpenAstroTracker-Desktop/assets/120117710/7a3da6dd-60a1-42b3-b229-70e30927fb53)

Here are the settings in OATCONTROL 

![image](https://github.com/OpenAstroTech/OpenAstroTracker-Desktop/assets/120117710/b0c8f1b6-2763-454e-9b52-6960f5bfcd22)

**Comment by ClutchplateDude** [comment]
Github issues are meant for specific bugs that have been discovered, not for troubleshooting, that's what our Discord server is for.

**Comment by ClutchplateDude** [comment]
However, I think this is actually an issue, so let's leave it open until a resolution is found.

**Comment by ClutchplateDude** [comment]
OK, found and fixed in OATControl V1.1.3.0. Thanks for reporting!
